lib/game.rb

```diff
@@ -0,0 +1,62 @@
+class Game
+  WIN_COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+  attr_accessor :player_1, :player_2, :board
+  attr_reader :winner
+
+  def initialize(player_1 = Human.new('X'), player_2 = Human.new('O'), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    @board.turn_count % 2 == 0 ? @player_1 : @player_2 
+  end
+
+  def over?
+    (@board.full? || won? || draw?) ? true : false
+  end
+
+  def won?
+    WIN_COMBINATIONS.each do |combination| 
+      if @board.get_cells(combination).uniq.length == 1
+        token = @board.cells[combination[0]]
+        if (token == "X") || (token == "O")
+          @winner = token
+          return true
+        end
+      end
+    end
+    false
+  end
+
+  def draw?
+    return true if @board.full? && (self.won? == false)
+    false
```

Comment:
This method always either returns `true` (if a certain condition is met) or `false`. Can you rewrite it to utilize the implicit return value of the boolean expression?